explore/src/main/scala/explore/Explore.scala

```diff
@@ -189,7 +189,7 @@ object ExploreMain extends IOApp.Simple {
       }
 
     (for {
-      dispatcher       <- Dispatcher[IO]
+      dispatcher       <- Dispatcher.parallel[IO]
```

Comment:
Cats Effect 3.4.0 introduces two new `Dispatcher` modes.
1. `Dispatcher.parallel`: creates a new fiber for each task and runs them in parallel
2. `Dispatcher.sequential`: runs tasks one-by-one on a single fiber

The "sequential" `Dispatcher` is far better optimized, because it does not have the overhead of a a fiber-per-task. But it is not safe to use it globally: while the `Dispatcher` is busy with one task (even if it is waiting for an async callback) it will be unable to run any other tasks.

Since these `Dispatcher`s are globally shared across many components, we can use the "parallel" `Dispatcher` for now. But in the long-term creating a "sequential" `Dispatcher` per-component can be more efficient.